libtcmu_common.h

```diff
@@ -154,7 +154,8 @@ int tcmu_cfgfs_dev_get_attr(struct tcmu_device *dev, const char *name);
 bool tcmu_cfgfs_file_is_supported(struct tcmu_device *dev, const char *name);
 int tcmu_cfgfs_exec_dev_action(struct tcmu_device *dev, const char *name,
 			       unsigned long val);
-int tcmu_set_dev_size(struct tcmu_device *dev);
+int tcmu_cfgfs_dev_set_ctrl_u64(struct tcmu_device *dev, const char *key,
```

Comment:
Should we name it as tcmu_cfgfs_set_dev_ctrl_u64 ?

Reply:
Lol, I think I misread your original comment. According to my comment above tcmu_cfgfs_exec_dev_action is misnamed and should be named tcmu_cfgfs_dev_exec_action to match the other functions. If that is what you were saying then I will fix that.

Reply:
Yep, I mean to ask for unifying them.

Reply:
@mikechristie along with tcmu_cfgfs_exec_dev_action, few other function violating the rule are:

tcmu_update_dev_lock_state()
tcmu_acquire_dev_lock()
tcmu_release_dev_lock()

Reply:
Those are for tcmu-runner only. In this pass we are only changing the libtcmu functions.

Reply:
Oh yeah, if those are exported to the lib then that is a bug.